tools/java/runfiles/BUILD

```diff
@@ -39,9 +39,18 @@ java_library(
         "Runfiles.java",
         "Util.java",
     ],
-    exported_plugins = [":auto_bazel_repository_processor"],
```

Comment:
Those shouldn't be removed, right?

Reply:
It doesn't really matter as this is `BUILD`, not `BUILD.tools`. I removed them as they were just unnecessary for the tests, but we can also keep them.

Reply:
SGTM